src/gauging.jl

```diff
@@ -209,22 +183,18 @@ end
 """Function to measure the 'canonicalness' of a state in the Vidal Gauge"""
 function vidal_itn_canonicalness(ψ::ITensorNetwork, bond_tensors::DataGraph)
```

Comment:
Maybe we can just make a single function `gauge_error`, i.e. `gauge_error(ψ::ITensorNetwork, bond_tensors::DataGraph)` for the Vidal gauge error and `gauge_error(ψ::ITensorNetwork)` for the symmetric gauge error.